buildenv/jenkins/JenkinsfileBase

```diff
@@ -479,13 +479,15 @@ def get_sources_with_authentication() {
 
 def get_sources() {
 	withEnv(['VERBOSE_CURL=SILENT']) {
-		if (params.CUSTOMIZED_SDK_URL_CREDENTIAL_ID) {
-			// USERNAME and PASSWORD reference with a withCredentials block will not be visible within job output
-			withCredentials([usernamePassword(credentialsId: "${params.CUSTOMIZED_SDK_URL_CREDENTIAL_ID}", usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+		dir(aqa-tests) {
```

Comment:
Should be `dir('aqa-test')`

Reply:
Fixed @sophia-guo

Reply:
Sorry, should be `dir('aqa-tests')`. The idea is in the aqa-tests directory to run the shell script.

Reply:
corrected @sophia-guo